linux-admin/part-2-tasks/task_YazanMajadba.md

```diff
@@ -0,0 +1,108 @@
+# First step on command line.
+1. Display your current directory. 
+pwd
+
+2. Change to the /etc directory.
+cd /etc
+
+3. Now change to your home directory using only three key presses.
+cd
+
+4. Go to the parent directory of the current directory.
+cd ..
+
+5. Go to the root directory.
+cd /
+
+6. List a long listing of the root directory using two ways.
+ls -l
+
+7. Using man pages list all files under /usr/bin and show list of files in long listing format,human readable size format,and show the oldest files first.
+ls  -lht /srt/bin
+
+8. Stay where you are, and list the contents of /bin and /sbin.
+ls /bin
+ls /sbin
+
+9. List the files in /boot in a human readable format.
+ls -hl
+
+10. Create in one command the directories ~/dir1/dir2/dir3 (dir3 is a subdirectory from dir2, and dir2 is a subdirectory from dir1 ).
+mkdir -p ~/dir1/dir2/dir3
+
+11. create /tmp/test-fld folder and /tmp/test-File file and move /tmp/test-File inside /tmp/test-fld. Now remove /tmp/test-fld using two ways.
+mkdir -p /tmp/test-fld/
+touch /tmp/test-File
+mv /tmp/test-File tmp/test-fld/
+rm /tmp/test-File
```

Comment:
`rm /tmp/test-file` will not work as you already moved the file inside `test-fld` also in order to remove the folder you need to use on the below commands
```bash
1) rm -r /tmp/test-fld (which will delete the directory alongside its content)
2) rm /tmp/test-fld/test-File; rmdir  /tmp/test-fld (delete the file first then delete the dir)```